src/dataexplorer/dataexplorer-application/src/components/PatientInfo/MyPatients.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { Row, Col } from 'antd';
+import Search from 'antd/lib/input/Search';
+const MyPatients = (props: any) => {
+    const handleSearch = (value: any) => {
```

Comment:
I would like to get some types here and not use any. Create interfaces or if there is just one type, use string, number, etc. This is something I would like for all components, functions, and states, otherwise, we lose the meaning of using typescript at all

Reply:
Added a few interfaces and defined the typing in the given componentfiles.